test/Summary.spec.js

```diff
@@ -1,55 +1,53 @@
 describe('The summary', function() {
     let page;
-
     before(async function() {
         page = await browser.newPage();
-        await page.goto('http://localhost:8080');
+        await page.goto(PAGE_URL);
     });
 
     after(async function() {
         await page.close();
     });
 
     describe('initially', function() {
-
         it("should not exist", async function() {
-            const summariescontainer = await page.$(SEL_SIDEBAR_CONTAINER);
-            expect(summariescontainer).to.be.null;
+            const summaryContainer = await page.$(SEL_SIDEBAR_CONTAINER);
+            expect(summaryContainer).to.be.null;
         });
     });
 
     describe('after submission', function() {
-
         before(async function() {
             await page.type(SEL_TEXTAREA, TEXT);
 
             const button = await page.$(SEL_SUBMIT);
             await button.click();
         });
 
-        it('should appear', async function() {
-            const summariescontainer = await page.$(SEL_SIDEBAR_CONTAINER);
-            expect(summariescontainer).to.exist;
+        it('should hide header component', async function() {
```

Comment:
Please use articles so the tests read as English sentences - "it should hide **_the_** header component"